pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -2408,6 +2409,59 @@ public void testMaxProducersPerTopicUnlimited() throws Exception {
         }
     }
 
+
+    @Test
+    public void testAcknowledgmentGroupSize() throws Exception {
```

Comment:
It does not seem to be related to admin. Can we move this test to the consumer related test?

Reply:
Fixed.
PTAL,thanks! @RobertIndie